src/main/java/com/inventorysystem/inventory/dao/RetrofitClientInstance.java

```diff
@@ -0,0 +1,22 @@
+package com.inventorysystem.inventory.dao;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RetrofitClientInstance {
+
+    private static Retrofit retrofit;
+    private static String BASE_URL = "link to the API";
```

Comment:
BASE_URL incomplete.

This looks like the start of a new feature.  Avoid adding new features in a code review, as a good code review should merge to master, and only features that meet the team's Definition of Done should be merged to master.  If functionality is introduced and not completed, it adds to technical debt, as it's unused code.